### ISSUE TITLE
No Updates Found Migrating to Winter on Non-Composer Install

### ISSUE THREAD

**Issue by NixFey** [wintercms/winter#120]
- Winter CMS Build: October build 471
- PHP Version: 7.4.3
- Database Engine: MySQL
- Plugins Installed: RainLab.Builder 1.0.26, RainLab.GoogleAnalytics 1.3.0, PanaKour.SendGrid 1.0.0
### Description:

I'm trying to migrate my October CMS installation to Winter using the updateServer method. I've set the updateServer in the cms.php to https://api.wintercms.com/marketplace, but when I check for updates in the UI or on the CLI with october:update, it says that no updates are available. I'm not sure if this is just an issue with my environment or maybe a bigger issue with the cms.php method of migrating away from October.

### Steps To Reproduce:

- Be on October CMS (non-composer install)
- Change updateServer to the Winter CMS one as described in migration instructions
- Have disableCoreUpdates false
- Check for updates on UI or CLI

**Comment by LukeTowers** [comment]
@evanlihou the update file isn't yet available from the marketplace API, should be available in a week at most.

**Comment by NixFey** [comment]
Thanks for the update @LukeTowers! If the marketplace API isn't yet distributing the new version, you might want to consider noting that in the [blog post](https://wintercms.com/blog/post/october-cms-you-know-it-dead).

I'm really excited to see where this project goes, and I'm glad to see the maintainers keeping a FOSS version alive. Keep up the good work!

**Comment by LukeTowers** [comment]
@evanlihou updated the blog post, thanks!

**Comment by zlobec** [comment]
@LukeTowers Is there any information about when the marketplace API will be available?

**Comment by LukeTowers** [comment]
@zlobec @bennothommo is currently working on the API for the core updates, hopefully soon. This month has been extremely busy for me as we just bought our first house so I've been running around like a chicken with its head cut off trying to deal with everything that entails and move.

**Comment by zlobec** [comment]
No worries ... I'm too in the middle of a move... and yeah... Keep up the good work.

**Comment by bennothommo** [comment]
We've all hit our roadblocks at the same time - mine is a grizzly newborn, hence why progress has been slow. I'm still slowly chipping away at the core updates - once that's done, the installer should be out shortly afterwards. I can't commit to a timeframe yet though, so I can only say "soon" as @LukeTowers said.

**Comment by graphixela** [comment]
Congrats on the move and the baby but am I missing something? It still says no updates...

**Comment by LukeTowers** [comment]
@graphixela nope, not missing anything. Priority for me this week is getting 1.1.4 out and then I can pick up work on the update delivery system through the marketplace again.

**Comment by graphixela** [comment]
Okay, thanks. It's marked as closed. A bit confusing.

**Comment by NixFey** [comment]
I closed this issue because my original question was answered, but maybe it should stay open until the API is actually implemented? @LukeTowers / @bennothommo is there another ticket that this is a duplicate of? If not feel free to open this back up and use for updates.

**Comment by NixFey** [comment]
Have there been any updates on this? It's starting to feel like this feature is never coming and I imagine it's quite important for people trying to migrate away from their October installations

**Comment by LukeTowers** [comment]
@evanlihou it is still coming, I've just been busy with personal stuff and client projects. All of that should be wrapping up in the next month so I should be able to crank it out in September / October. In the meantime, there's always the composer route for people who need to swap over immediately.

**Comment by NixFey** [comment]
Just commenting to keep this open

**Comment by fpomidoro** [comment]
hello all, is there any news or ETA on this? thanks!

**Comment by LukeTowers** [comment]
It's actively being worked on @fpomidoro, see #438

**Comment by fpomidoro** [comment]
Just keeping this open! Thanks!

**Comment by fpomidoro** [comment]
Keeping it open. Thanks!

**Comment by fpomidoro** [comment]
Comment to keep it open, any updates and/or ETA? Thanks!

**Comment by LukeTowers** [comment]
Still hard at work on it @fpomidoro, should be ready in the next month or so we're hoping. Working on a refresh of our main website (both design and infrastructure) and a lot of behind the scenes organizational things at the same time.

**Comment by NixFey** [comment]
If the devs aren't planning on implementing this feel free to close but just in case I'll keep it open.

I know priorities are constantly pulling in different directions, but I do hope this does get done eventually :)

**Comment by bennothommo** [comment]
We're definitely working on it. We're all just very, very busy at the moment, so it's not as fast as we'd like. :(

**Comment by fpomidoro** [comment]
Hi! Just checking if there is any clear roadmap/date on this? Thanks!

**Comment by LukeTowers** [comment]
We're working on a necessary refactor to the entire plugin management / update management system right now (https://github.com/wintercms/winter/pull/1259) after which we'll be able to work on the update delivery system. For now, the best option is still just composer.
